js_modules/dagit/packages/core/src/CapturedLogPanel.tsx

```diff
@@ -121,34 +113,35 @@ const initialState: State = {
 
 const CapturedLogSubscription: React.FC<{
   logKey: string[];
-  onLogData: (logData: CapturedLogsSubscription_capturedLogs) => void;
+  onLogData: (logData: CapturedLogFragment) => void;
 }> = React.memo(({logKey, onLogData}) => {
-  useSubscription<CapturedLogsSubscription, CapturedLogsSubscriptionVariables>(
-    CAPTURED_LOGS_SUBSCRIPTION,
-    {
-      fetchPolicy: 'no-cache',
-      variables: {logKey},
-      onSubscriptionData: ({subscriptionData}) => {
-        if (subscriptionData.data?.capturedLogs) {
-          onLogData(subscriptionData.data.capturedLogs);
-        }
-      },
+  useSubscription(CAPTURED_LOGS_SUBSCRIPTION, {
+    fetchPolicy: 'no-cache',
+    variables: {logKey},
+    onSubscriptionData: ({subscriptionData}) => {
+      if (subscriptionData.data?.capturedLogs) {
+        onLogData(subscriptionData.data.capturedLogs);
+      }
     },
-  );
+  });
   return null;
 });
 
-const CAPTURED_LOGS_SUBSCRIPTION = gql`
+const CAPTURED_LOGS_SUBSCRIPTION = graphql(`
   subscription CapturedLogsSubscription($logKey: [String!]!, $cursor: String) {
     capturedLogs(logKey: $logKey, cursor: $cursor) {
-      stdout
-      stderr
-      cursor
+      ...CapturedLog
     }
   }
-`;
 
-const CAPTURED_LOGS_METADATA_QUERY = gql`
+  fragment CapturedLog on CapturedLogs {
```

Comment:
Oh it's very cool that these don't have to be in their own gql strings / tags. (Maybe they never did, but I don't think I've ever one defined one inline like this!)